tox.ini

```diff
@@ -66,6 +66,14 @@ commands =
     pylint --rcfile=pylintrc --output-format=parseable {toxinidir}/src/pymovements {toxinidir}/tests
 
 
+[testenv:mypy]
+deps =
+    mypy
+commands =
+    mypy src
```

Comment:
use a more safe style by using `{toxinidir}`to be sure to use the correct tox environment 
including `{posargs}` will let you pass additional positional arguments to mypy.

example command defintion for flake8:

https://github.com/aeye-lab/pymovements/blob/07f13f60b8b0669ccb5eeda903b0b48d9a93843d/tox.ini#L55-L56